src/PIL/ImageQt.py

```diff
@@ -155,11 +160,11 @@ def _toqclass_helper(im: Image.Image | str | QByteArray) -> dict[str, Any]:
         assert palette is not None
         colortable = [rgb(*palette[i : i + 3]) for i in range(0, len(palette), 3)]
     elif im.mode == "RGB":
-        # Populate the 4th channel with 255
+        # Populate the 4th channel with 255 #!
         im = im.convert("RGBA")
 
         data = im.tobytes("raw", "BGRA")
-        format = getattr(qt_format, "Format_RGB32")
+        format = getattr(qt_format, "Format_ARGB32") # 4th channel was populated in line 164 
```

Comment:
No need to repeat an earlier comment, and the line number will go out of sync:
```suggestion
        format = getattr(qt_format, "Format_ARGB32")
```